src/common/containers/landing.js

```diff
@@ -42,9 +42,16 @@ class Landing extends Component {
               </ul>
 
               <div className={ styles.bannerButton }>
-                <Anchor href="/auth/authenticate" icon={ octocat } flavour='embiggened' appearance='positive' >
-                  Set up in minutes
-                </Anchor>
+                { !this.props.auth.authenticated &&
+                  <Anchor href="/auth/authenticate" icon={ octocat } flavour='embiggened' appearance='positive' >
+                    Set up in minutes
+                  </Anchor>
+                }
+                { this.props.auth.authenticated &&
+                  <div>
+                    Hi { this.props.user.name }, <a href="/dashboard">let’s check out your repos</a>.
```

Comment:
This will be prone to the same issue we solve in #239

`user.name` may be empty in GH, so we need to fall back to `user.login` in such case.